#! /bin/sh /usr/share/dpatch/dpatch-run
## 01_manpage.dpatch by Mario Iseli <admin@marioiseli.com>
##
## DP: Fixes the line 195 in the manpage because lintian treated it as wrong
## DP: linebreaking due to the many "\-" in a chain.

@DPATCH@

diff -Naur mcabber-0.9.3/doc/mcabber.1 mcabber-0.9.3.new/doc/mcabber.1
--- mcabber-0.9.3/doc/mcabber.1	2007-06-20 18:05:40.000000000 +0100
+++ mcabber-0.9.3.new/doc/mcabber.1	2007-06-20 18:07:44.000000000 +0100
@@ -192,7 +192,7 @@
 Ctrl\-o
 .sp
 T}:T{
-Accept line and put the next history line in the input line (accept\-line\-and\-down\-history)
+Accept line and put the next history line in the input line (accept \- line \- and \- down \- history)
 .sp
 T}
 T{
